infra/base-images/base-runner/Dockerfile

```diff
@@ -79,10 +79,5 @@ ENV GOPATH /root/go
 # $GOPATH/bin is for the binaries from the dependencies installed via "go get".
 ENV PATH $PATH:/root/.go/bin:$GOPATH/bin
 
-# gocovmerge merges coverage profiles.
-RUN go get -u github.com/wadey/gocovmerge
-# pprof-merge merges performance profiles.
-RUN go get -u github.com/rakyll/pprof-merge
-# gocovsum produces a json summary.
-RUN go get -u github.com/catenacyber/gocovsum
-
+# Set up Golang coverage modules.
+RUN go get github.com/google/oss-fuzz/infra/go/coverage/...
```

Comment:
I guess it doesn't matter if we never change these scripts, but this looks like it will be a nightmare to debug since you can change the files locally, but this dockerfile will still fetch the old versions from github.com

Reply:
How would you do it ?

Reply:
It's possible that you can use the Docker `COPY` command, but we'd have to make sure these images are built using the root of OSS-Fuzz as the working directory.

Reply:
Yes... I guess next change to the scripts will come with a change to the Dockerfile then...

Reply:
ACK :-)